packages/fluentui/react-northstar/src/utils/date-time-utilities/dateFormatting/formatYear.test.ts

```diff
@@ -0,0 +1,24 @@
+import * as DateGrid from './formatYear';
```

Comment:
react-northstar unit tests execute tests from `test/.*-test`:
https://github.com/microsoft/fluentui/blob/50c2a17463c06292662904065cbeba1ee05122a1/scripts/jest/index.js#L10
That means that none of the moved tests are actually run.
You need to rename and move the test files.

Reply:
Done